README.md

```diff
@@ -118,3 +107,67 @@ You *can* instead adjust `tick.html` and `tick.js` to run the WebSocket on
 a separate port, but then you'll also need to adjust both `Dockerfile` and
 `docker-compose.yml` to expose that extra port, along with possibly needing
 to allow that extra port through any firewall you have.
+## API Reference
+
+### Web Socket API
+As part of providing the main web page `tick.js` provides a Web Socket
+which will receive a `'message'` when there is a new tick.
+
+This socket is provided on the standard HTTP/HTTPS port, using whichever URI
```

Comment:
I know this is *my* wording, but technically it has to be using the `ws(s)` protocol.  I seem to recall that trying to fake this in curl (without websocket support) by setting the appropriate extra headers Just Did Not Work.

It's just that the host and port aspects of the URI are indeed the same as the HTTP(s) ones.

Reply:
```suggestion
This socket is provided over the `ws(s)` protocol on the standard HTTP/HTTPS port, using the same URI that
```

Reply:
Actually I'll fix this bit up after merge, as there are some other subtleties, depending on what you're using to make the websocket connection.